src/utils/getCountOfCompletedTodos.ts

```diff
@@ -0,0 +1,13 @@
+import { Todo } from '../types/Todo';
+
+export function getCountOfCompletedTodos(todos: Todo[]): number {
+  const countOfActiveTodos = todos.reduce((acc, todo) => {
+    if (todo.completed) {
+      return acc + 1;
+    }
+
+    return acc;
+  }, 0);
+
+  return countOfActiveTodos;
```

Comment:
It can be done simplier with `filter` and `length`

Reply:
> It can be done simplier with `filter` and `length`

Not fixed, do it in the next task